index.html

```diff
@@ -48,13 +60,97 @@
+    <ul>
+      <li>Cassiopia, the human shadow sorcerer who got seperated from her 
+        family while trying to access the Radiant Citadel.
+      <br><a href="https://www.dndbeyond.com/characters/109114519">Cassiopia's character sheet</a><br></li>
+      <li>Estelle, the human bard who is on a quest to expand her lifespan to
+        live as long as her adoptive dwarf family.
+      <br><a href="https://www.dndbeyond.com/characters/109187761">Estelle's character sheet</a><br></li>
+      <li>Gertrude, a gnomish order cleric who was once a thief who
+        has now unwillingly chaged their ways due to the control
+        of a god in her head
+      <br><a href="https://www.dndbeyond.com/characters/109648959">Gertrude's character sheet</a><br></li>
+      <li>Lin, a human fighter who's family sent him away when he 
+        was really young and now he's on a quest to reconnect
+      <br><a href="https://www.dndbeyond.com/characters/109087827">Lin's character sheet</a><br></li>
+      <li>Okihtcita, a goliath ranger who is hunting an illusive
+        monster and wants to kill them with the axes that his parents 
+        gifted him on his deathbed
+      <br><a href="https://www.dndbeyond.com/characters/109068678">Okihtcita's character sheet</a><br></li>
+      <li>Tieran, a half-elf warlock who was given a mission from
+        his angel patron to investigate the Radiant Citadel
+      <br><a href="https://www.dndbeyond.com/characters/109060064">Tieran's character sheet</a><br></li>
+    </ul>
+    <h5>Give me suggestions!</h5>
+    <p>I'm not very good at coming up with things to put in my game
+      so I'm giving all my followers the ability to shape my game!
+    </p>
+    <form action="I'mnottakinganyone'ssuggestions" method="post">
+      <ul>
+        <li>
+          <label for="suggestione">Suggestion:</label>
+          <textarea rows="5" id="suggestione" placeholder="Give Okihtcita a magic item"></textarea>
+        </li>
+        <li>
+          <label for="suggestifive">Next magic item?:</label>
+          <input type="text" id="suggestifive" placeholder="bag of holding"/>
+        </li>
+        <li>
+          <label for="suggestitwo">Should I kill a player?:</label>
+          <input type="text" id="suggestitwo" placeholder="y/n"/>
+        </li>
+        <li>
+          <label for="suggestithree">Name of suggester:</label>
+          <input type="text" id="suggestithree" placeholder="Galland Thunderhammer"/>
+        </li>
+        <li>
+          <label for="suggestifour">email of suggester:</label>
+          <input type="text" id="suggestifour" placeholder="thundersmith@neverwinter.ca"/>
+        </li>
+        <li>
+          <button type="submit">Submit</button>
+        </li>
+      </ul>
+    </form>
   </body>
+  <footer>
+    <h2>The Gelatinous Blog!</h2>
+    <img src="gelatiousblog.png" alt="blog logo"/>
+    
+    <nav>
+      <ul>
+        <li><a href="Maps.html">Maps</a></li>
```

Comment:
My recommendation here is to add an `h3` at the start of `nav` elements. Check out the tutorial exercises for code samples